docs/sphinx/sections/api/apidocs/metadata.rst

```diff
@@ -65,3 +65,12 @@ These APIs provide the ability to express column schemas (`TableSchema`), rows/r
 .. autoclass:: TableColumnLineage
 
 .. autoclass:: TableColumnDep
+
+Metadata functions
+^^^^^^^^^^^^^^^^^^
+
+These APIs are used to attach additional metadata to certain objects in the Dagster ecosystem.
```

Comment:
> certain objects

Isn't it just assets? This feels vaguely cryptic 👻 

Also, could we add a link to the relevant guide (or guides) here?

Reply:
Updated